converters/toZigbee.js

```diff
@@ -2057,7 +2059,7 @@ const converters = {
     danfoss_algorithm_scale_factor: {
         key: ['algorithm_scale_factor'],
         convertSet: async (entity, key, value, meta) => {
-            const payload = {'danfossAlgorithmScaleFactor': {value: value, type: 0x20}};
+            const payload = {'danfossAlgorithmScaleFactor': value};
             await entity.write('hvacThermostat', payload, manufacturerOptions.danfoss);
```

Comment:
Add a return here to be consistent with the other ones?

Reply:
Done
https://github.com/Koenkk/zigbee-herdsman-converters/pull/2592/commits/aa227ca98224cb5f5700c0755db6f481982b9b8f